drape_frontend/message_queue.cpp

```diff
@@ -80,6 +80,19 @@ void MessageQueue::PushMessage(drape_ptr<Message> && message, MessagePriority pr
   CancelWaitImpl();
 }
 
+void MessageQueue::FilterMessages(TFilterMessageFn needFilterMessageFn)
+{
+  lock_guard<mutex> lock(m_mutex);
```

Comment:
How about to:
```
if (!needFilterMessageFn)
  return;
```
?
The function is in the public section, so may be called from any place.